x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -1236,6 +1276,55 @@ export class RulesClient {
     }
   }
 
+  private async logInstanceEvent(
```

Comment:
Rather than copy the code from wherever else we do this - I believe the task runner - it would be nice if we could reuse that task runner code, so make sure we get all the same fields get populated.  Probably means extracting the code that builds the event docs into a separate module so it can be reused.  Otherwise if we change the data written in one place, we'll need to remember to update it in the other - tech debt.  Could be done later though, via a separate issue/PR.